content/lxd/advanced-guide.md

```diff
@@ -56,6 +56,9 @@ Usage:
 
 	lxc launch imageserver:imagename instancename -p profile1 -c key1=value
 
+!!! note
+    You can apply the `-p` option more than once. The profiles will be combined then.
+
```

Comment:
This is already stated in the next note (line 62/65). Maybe update that wording of that note if you think it's not clear enough?

Reply:
Oh wow! :-)

I completely missed that "text note", as after the first example, I followed the "See Profiles below for details." link which jumped over the "text note".

Instead, maybe we could remove the "Note: See Profiles below for details."? As Profiles is next anyway?

Or at least move it to the bottom of the section, though then it would be just above the "Profiles" section.

Reply:
I agree - the note linking to Profiles is out of place, and not really needed here.
Removing that note should make things clearer. :)

Reply:
:thinking: Initially I wanted to make it clear where to find more information regarding profiles, because at this stage, people might not know what profiles are.

I guess we could remove it, or just add it as a normal sentence and not as a note.

Reply:
Thank your for taking the time to clarify this!

I addressed your feedback.